src/components/Class/ClassCard/styles.scss

```diff
@@ -0,0 +1,46 @@
+.class-cards {
+  width: 380px;
+  background: linear-gradient(to bottom left, #83b344 20%, #ffc39e 100%);
+  height: auto;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 25px;
+  margin-left: 32px;
+}
+
+.card-content {
```

Comment:
the better way to name the class child could be something like this. 
```

.class-cards >.card-content {
  padding: 25px;
  font-family: "Times New Roman", Times, serif;
  text-transform: capitalize;
}
```
that we this class name won't affect any other div with the same class name. Let me know if you need more clarity on it.:)

Reply:
done